allenact_plugins/manipulathor_plugin/manipulathor_environment.py

```diff
@@ -95,6 +147,20 @@ def __init__(
             **prepare_locals_for_super(locals())
         )
 
+        results = pd.read_csv("allenact_plugins/manipulathor_plugin/vibrations.csv")
+        # use dict is much faster to query than dataframe
+        self.vibration_distances = {}
+        for i in range(results.shape[0]):
+            self.vibration_distances[
+                results.at[i, "scene"] + "-" + results.at[i, "object"]
+            ] = results.at[i, "dist"]
+
+        raw = (
+            results.groupby("scene").sum() / 200
+        )  # averge vibration per step on all the objects
+        raw = raw.clip(lower=0.001)
+        self.vibration_distances_scene = raw.to_dict()["dist"]
```

Comment:
These vibrations seem like they are task-specific, can we move this to the other code-base?

Reply:
Make sense, it is only used in `DisturbanceSensor`, which should be also moved to my stand-alone repo.